feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -171,7 +170,7 @@ func getRespCode(args *testArgs) (int32, error) {
 }
 
 // getGDPParameter returns GDP related parameters for testPacketIn testcase.
-func getGDPParameter(t *testing.T) PacketIO {
+func getGDPParameter(_ *testing.T) PacketIO {
```

Comment:
Can you just remove the parameter entirely and while you're at it, drop the word "get" from the method name to conform to go style? The name "gdpParameter()" is fine. There appears to only be one caller.

Reply:
I think we can remove this func altogether, moved it inline.